packages/cli/src/workflows/workflow.service.ee.ts

```diff
@@ -329,8 +331,7 @@ export class EnterpriseWorkflowService {
 		}
 
 		// 6. deactivate workflow if necessary
-		const wasActive = workflow.activeVersionId !== null;
-		if (wasActive) {
+		if (workflow.activeVersionId !== null) {
```

Comment:
Why are we reverting these? I don't think that's needed for this change? Also IMHO the previous one was clearer